### PR TITLE
Update stack version to 7.13

### PR DESCRIPTION
**Pull request by pebrc**
Fixes https://github.com/elastic/cloud-on-k8s/issues/4518

### CODE REVIEW

File: .ci/pipelines/e2e-tests-stack-versions-gke.Jenkinsfile [hunk 1]
```diff
@@ -134,14 +134,25 @@ pipeline {
...
+                    }
+                }
+                stage("7.13.0") {
+                    agent {
+                        label 'linux'
+                    }
+                    steps {
+                        unstash "source"
+                        script {
+                            runWith(lib, failedTests, "eck-713-${BUILD_NUMBER}-e2e", "7.13.0")
```
Comment (thbkrkr) [inline .ci/pipelines/e2e-tests-stack-versions-gke.Jenkinsfile:155]: The declaration to delete the cluster is missing:
```diff
diff --git a/.ci/pipelines/e2e-tests-stack-versions-gke.Jenkinsfile b/.ci/pipelines/e2e-tests-stack-versions-gke.Jenkinsfile
index 28e38446..ff77351b 100644
--- a/.ci/pipelines/e2e-tests-stack-versions-gke.Jenkinsfile
+++ b/.ci/pipelines/e2e-tests-stack-versions-gke.Jenkinsfile
@@ -194,7 +194,8 @@ pipeline {
                     "eck-79-${BUILD_NUMBER}-e2e",
                     "eck-710-${BUILD_NUMBER}-e2e",
                     "eck-711-${BUILD_NUMBER}-e2e",
-                    "eck-712-${BUILD_NUMBER}-e2e"
+                    "eck-712-${BUILD_NUMBER}-e2e",
+                    "eck-713-${BUILD_NUMBER}-e2e"
```

Comment (pebrc) [reply .ci/pipelines/e2e-tests-stack-versions-gke.Jenkinsfile:155]: 👍

File: .ci/pipelines/e2e-tests-stack-versions-gke.Jenkinsfile [hunk 2]
```diff
@@ -184,6 +195,7 @@ pipeline {
                     "eck-710-${BUILD_NUMBER}-e2e",
                     "eck-711-${BUILD_NUMBER}-e2e",
                     "eck-712-${BUILD_NUMBER}-e2e"
```
Comment (thbkrkr) [inline .ci/pipelines/e2e-tests-stack-versions-gke.Jenkinsfile:197]:
```suggestion
                    "eck-712-${BUILD_NUMBER}-e2e",
```

**Review by thbkrkr — APPROVED** [review]
:+1:

**Pull request merged**